QUICK_START.md

```diff
@@ -148,6 +148,8 @@ returned by the "startup" call.  You can click on the value of
 "short_pitch" in the body of the "startup patch" call and then click
 change to change it to any other value.
 
+It should be noted that due to client, browser, or firewall restrictions, it may be necessary to send PATCH calls as POSTS and override them.  The middleware which overrides these types of POSTs is included in the code base.
```

Comment:
the readme should make mention of how to do the override (providing the name of the header to pass and what the value should be)

Reply:
Ah, ok.